demos/python_demos/multi_camera_multi_target_tracking/README.md

```diff
@@ -90,32 +90,32 @@ optional arguments:
   -u UTILIZATION_MONITORS, --utilization_monitors UTILIZATION_MONITORS
                         Optional. List of monitors to show initially.
 ```
-Minimum command examples to run the demo:
+Minimum command examples to run the demo for person tracking (for vehicle tracking the commands are the same with appropriate vehicle detection/re-identification models):
 
 ```
 # videos
-python multi_camera_multi_person_tracking.py \
+python multi_camera_multi_target_tracking.py \
     -i path/to/video_1.avi path/to/video_2.avi \
     --m_detector path/to/person-detection-retail-0013.xml \
     --m_reid path/to/person-reidentification-retail-0103.xml \
-    --config config.py
+    --config configs/person.py
```

Comment:
You should update a default value for `--config`.

Reply:
Fixed